Assets/BossRoom/Scripts/Client/UI/ConnectionStatusMessageUIManager.cs

```diff
@@ -38,13 +38,17 @@ void OnConnectStatus(ConnectStatus status)
                     PopupPanel.ShowPopupPanel("Connection Failed", "The Host is full and cannot accept any additional connections.");
                     break;
                 case ConnectStatus.Success:
+                    PopupPanel.ShowPopupPanel("Success!", "Joining Now");
                     break;
                 case ConnectStatus.LoggedInAgain:
                     PopupPanel.ShowPopupPanel("Connection Failed", "You have logged in elsewhere using the same account.");
                     break;
                 case ConnectStatus.GenericDisconnect:
                     PopupPanel.ShowPopupPanel("Disconnected From Host", "The connection to the host was lost");
                     break;
+                case ConnectStatus.Reconnecting:
+                    PopupPanel.ShowPopupPanel("Attempting reconnection", "Lost connection to the Host, attempting to reconnect...");
```

Comment:
same here, this should be a temporary popup, not something that requires an interraction.